groups/index.html

```diff
@@ -36,6 +36,13 @@ <h3 class="not-verified-tag hidden">
           <ul class="groups-list"></ul>
         </aside>
         <main>
+          <div class="dropdown hidden" id="dropdown_container">
+            <button class="task_filter-button" id="sortby_text">Sort by</button>
+            <div id="dropdown_main" class="dropdown-content">
+              <a data-list="1">Popular with dev</a>
```

Comment:
There can be users who aren't developers, so not sure about this. Please confirm with Ankush/Tejas if hasn't been.

Reply:
This UI is approved [design doc](https://docs.google.com/document/d/1SqHcsh-KCBewySP1rCx8k2AJqfdsQJMaXjAxMkHUrKY/edit#heading=h.cxpij3s7de98)

Reply:
Changes the text after suggestion from tejas.